packages/std/src/results/cosmos_msg.rs

```diff
@@ -120,10 +120,10 @@ pub enum DistributionMsg {
     },
 }
 
-fn binary_to_string(data: &Binary, fmt: &mut std::fmt::Formatter) -> Result<(), std::fmt::Error> {
-    match std::str::from_utf8(data.as_slice()) {
+fn binary_to_string(data: &Binary, fmt: &mut core::fmt::Formatter) -> core::fmt::Result {
+    match core::str::from_utf8(data.as_slice()) {
         Ok(s) => fmt.write_str(s),
-        Err(_) => write!(fmt, "{data:?}"),
+        Err(_) => core::fmt::Debug::fmt(data, fmt),
```

Comment:
nit: `core::fmt` is imported above
```suggestion
        Err(_) => fmt::Debug::fmt(data, fmt),
```